orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -622,6 +647,43 @@ def __setstate__(self, state):
 
         _utils.reinitialize_graph_execution_manager(self)
 
+    def _check_embedding_sparsity(self):
```

Comment:
move this into runter_inspector?

Reply:
This is not convenient to move into runtime_inspector because the runtime_inspector.input_density_ob is not initialized before export_model. Need more refactor.  I will move it into runtime_inspector in next pr which will do the loss label inspect automatically.